libsakura/cmake-modules/SetArchFlags.cmake

```diff
@@ -41,5 +41,8 @@ macro(set_cxx_flags_from_arch SIMD_ARCH)
 	elseif("${SIMD_ARCH}" STREQUAL "SSE4")
 		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${DefaultArch}")
 		set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${DefaultArch}")
+	elseif(SIMD_ARCH STREQUAL "ARM")
+		set(CMAKE_CXX_FLAGS "-Wall -Wno-deprecated-register -fPIC -pipe-march=armv8-a")
```

Comment:
Do you need a whitespace after `-pipe`, i.e. `-pipe -march=armv8-a`?

Reply:
Yeah was a typo from copy pasting. Fixed.